homeassistant/components/stookwijzer/sensor.py

```diff
@@ -63,3 +64,14 @@ def native_value(self) -> str | None:
         if self._client.state is None:
             return None
         return StookwijzerState(self._client.state).value
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any] | None:
```

Comment:
Can we maybe create separate entities for these attributes instead?

Reply:
Right now, I followed the same approach as Buienradar has.

Reply:
With newer changes we are trying to separate more sensors instead of using attributes

Reply:
Thanks for your review. If you have time to create the separate sensors, please feel free to make changes. I don't have this many time available now unfortunately.

Reply:
Can we omit them for this PR?

Reply:
More sensors means an update coordinator is needed as well. So this will consume more time than I can spend right now. Maybe if you have some time left, you can work on this?

Reply:
I am currently also busy, but I am happy to help out. But for this PR to merge I would like to at least remove the extra state attributes, otherwise we're adding stuff people will rely on which we will remove in the near future